src/aicourse/agent.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, qfunc, env):
         self.env = env
     
     @torch.no_grad()
-    def get_action(self, observations, validation=False, eps=0.):
```

Comment:
This is not needed, as in `play.py` we have `validation=True` which blocks the stochastic effect of `eps`.

Reply:
Thanks for your comment. I got it 😃.